core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -795,7 +795,12 @@ abstract static class ProcfsUnix extends Unix {
     /**
      * A process.
      */
-    public abstract class UnixProcess extends OSProcess {
+    public abstract class UnixProcess extends OSProcess implements Serializable {
```

Comment:
`OSProcess` is already `Serializable`.
```suggestion
    public abstract class UnixProcess extends OSProcess {
```

Reply:
Oh sorry, that should not have happened